lab_4_retrieval_w_clustering/main.py

```diff
@@ -170,6 +240,8 @@ def __init__(self, db: DocumentVectorDB) -> None:
         Args:
             db (DocumentVectorDB): Object of DocumentVectorDB class.
         """
+        super().__init__(self._vectorizer, self._tokenizer)
```

Comment:
i think you should put here `db.get_vectorizer()`? No?

Reply:
got it